sim/core/test_utils.go

```diff
@@ -34,7 +34,7 @@ const LongDuration = 300
 var DefaultTargetProto = &proto.Target{
 	Level: CharacterLevel + 3,
 	Stats: stats.Stats{
-		stats.Armor:       11977,
+		stats.Armor:       24835,
 		stats.AttackPower: 650,
```

Comment:
I would set the default AP to 0, even if the real value for bosses is non-zero, since that lets the `minBaseDamage` field directly specify the minimum swing damage without needing to do extra math on the AP scaling. Since Cata, there are no more debuffs that subtract the target's AP, so it doesn't matter what the true value is anymore as long as damage parameters are fit consistently.